From 7be3393ed5883c5f56abb2979485d163b6483616 Mon Sep 17 00:00:00 2001
From: Zubair Lutfullah Kakakhel <Zubair.Kakakhel@imgtec.com>
Date: Fri, 12 Feb 2016 11:04:40 +0000
Subject: perf build: Add EXTRA_LDFLAGS option to makefile

To compile for little-endian systems, you need to pass -EL to CC
and LD.

EXTRA_CFLAGS works to pass -EL to CC.
Add EXTRA_LDFLAGS to pass -EL to LD.

Signed-off-by: Zubair Lutfullah Kakakhel <Zubair.Kakakhel@imgtec.com>

---
Raised by Sean. Build error where perf would be compiled for LE.
But be linked for BE.

Bug in makefile upstream.

Upstream link https://lkml.org/lkml/2016/2/9/330

Change-Id: Id3e0a6ff83eb5c7f330bc901ed0ab1101b5cdc41
---
 tools/perf/Makefile.perf | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/perf/Makefile.perf b/tools/perf/Makefile.perf
index 0d19d54..fb45fad 100644
--- a/tools/perf/Makefile.perf
+++ b/tools/perf/Makefile.perf
@@ -135,6 +135,8 @@ $(call allow-override,LD,$(CROSS_COMPILE)ld)
 
 PKG_CONFIG = $(CROSS_COMPILE)pkg-config
 
+LD += $(EXTRA_LDFLAGS)
+
 RM      = rm -f
 LN      = ln -f
 MKDIR   = mkdir
-- 
1.9.1

